vhdbuilder/packer/install-dependencies.sh

```diff
@@ -314,46 +315,8 @@ done
 # v1.21.1
```

Comment:
we can also remove these comments, even better let's add them as comment field to the json file

Reply:
Talked to @andyliuliming, we don't want to download all versions since we have some custom logic per version thus having those configs in separate file is more suitable for this scenario